diffpy/pdfmorph/pdfmorphapp.py

```diff
@@ -62,9 +61,27 @@ def custom_error(self, msg):
     parser.add_option(
         '-s',
         '--save',
-        metavar="FILE",
+        metavar="SFILE",
         dest="savefile",
-        help="Save manipulated PDF from FILE1 to FILE. Use \'-\' for stdout.",
+        help="Save manipulated PDF from FILE1 to SFILE. Use \'-\' for stdout.",
+    )
+    parser.add_option(
+        '--sequence',
+        dest="sequence",
+        action="store_true",
+        help=f"""Changes usage to \'{prog_short} [options] FILE DIRECTORY\'. FILE
+will be morphed with each file in DIRECTORY as target.
+Files in DIRECTORY are sorted by alphabetical order unless
+--temperature is enabled. Plotting and saving options are for
+a Rw plot and table respectively.""",
+    )
+    parser.add_option(
```

Comment:
this is still really sorting by filename here so I think this is not needed.  

Another option would be to really store by temperature where temperature is piece of metadata.  Then we would need to know where the metadata came from.  The gr files allow metadata in the header region so we could parse the file for a "temperature" field and then use the value of that item in alpha-numeric order.  If we offer that, we could allow sorting by any metadata field, so allow the option for a user to specify a string that matches any field in the header and then sort alphanumercically by the value of those items.   This seems better to me.  What do you think?

I guess default behavior is generally to make the higher-T PDF the target and the lower-T the morph (it is easier to broaden than sharpen data with morphs).  This may frustrate a user who has some fancy sorting that is not temperature, like a battery discharging or something and it is becoming less ordered in time or sthg, or for some other reason, so we could offer a `--reverse` option so the sorting is done with the higher value morphing onto a lower value target.

Many users might have a table in their notebooks or in a csv file that does a mapping of "temperature" onto the filename (or any other item-key onto filename).  Another approach would be to have tools that can parse such a file and collect the metadata that way.  Moving forwards we would like to move in the direction of databases, so being able to parse the same info from json serialized data also would be useful.   In this world, we would allow the user to specify a file (support csv or json or yaml formats?) that maps metadata fields to a filename.  Then this could be the target we use for our sorting.   If we had tools that could take this information and inject it into .gr files as an option, or vice-versa, parse out the gr file headers into metadata files, it would be good.   This would then be very helpful in general. 

With this in mind, I would suggest that we have a ``--sort-by`` that takes a string which is the field we want to sort by, a `--reverse` which reverses the sort sense when true.  Your `--temperature` option could be omitted, or could be there but is a helper meaning "sort-by temperature".  By default it could look for `medatada..<ext>` as the metadata filename where <ext> could be `["csv", "json", "yaml", "yml"]` or sthg, but also have a `--metadata-filename` option that overrides this.   What do you think?

These issues are way broader than pdfmorph, so we could add a bunch of capabilities in `diffpy.utils` that do the file handling, metadata serialization and deserialization in different formats so these things are available across all of diffpy (and beyond).  That could be a separate little mini project.